CONTRIBUTING.rst

```diff
@@ -109,14 +108,14 @@ a new pull request against the `corresponding docs section of the MLflow code ba
 
 For more information about Plugins, see https://mlflow.org/docs/latest/plugins.html.
 
-Developing and testing changes to MLflow
-########################################
+Developing and testing MLflow
```

Comment:
IMO the original wording here is actually a bit clearer, is there a reason to change it? With "Developing and testing MLflow" it's a bit less clear that we're testing changes to MLflow, as opposed to using MLflow to develop another project

Reply:
One argument would be while the proposition "to" or "for" makes it specific to MLflow, we are already talking about contributions to MLFlow. The document is about contributing to MLflow. So it may be just redundant. However, the CONTRIBUTIONS.RST was for MLflow and say, Spark MLlib, then having that specificity would make it more clear.

Reply:
Gotcha, yeah seems fine given that we explicitly state that this page is for "contributing to MLflow" at the top